sw/device/tests/rom_ext/rom_ext_parser_unittest.cc

```diff
@@ -226,5 +227,36 @@ TEST_F(ExtensionGetTest, Success) {
   EXPECT_THAT(ext3, Eq(ext3_expected));
 }
 
+class EntryGetTest : public ParserTest {};
+
+TEST_F(EntryGetTest, Success) {
+  EXPECT_EQ(rom_ext_get_entry(params_),
+            kRomExtManifestSlotA + ROM_EXT_ENTRY_POINT_OFFSET);
+}
+
+class InterruptVectorGetTest : public ParserTest {};
+
+TEST_F(InterruptVectorGetTest, Success) {
+  EXPECT_EQ(rom_ext_get_interrupt_vector(params_),
+            kRomExtManifestSlotA + ROM_EXT_INTERRUPT_VECTOR_OFFSET + 0x1);
```

Comment:
nit: This seems redundant.

Reply:
Why `+1`? could you document the line please, or use a named define/constant.

Reply:
I'm going to delete this in favour of the test below, which is clearer.

Reply:
Removed.